azure/durable_functions/tasks/new_uuid.py

```diff
@@ -1,6 +1,11 @@
 from uuid import uuid5, NAMESPACE_OID
 
 from azure.durable_functions.constants import DATETIME_STRING_FORMAT
+import typing
+
+if typing.TYPE_CHECKING:
```

Comment:
Just to clarify, this makes us only import when type checking to avoid circular references?

Reply:
Indeed!